### ISSUE TITLE
Order of labels impacts detection confidence scores

### ISSUE THREAD

**Issue by saibhamidipati** [urchade/GLiNER#192]
I'm using the PII model using the following script:

```
model = GLiNER.from_pretrained("urchade/gliner_multi_pii-v1")
model.eval()
entities = model.predict_entities(text, labels)

for entity in entities:
    print(entity["text"], "=>", entity["label"], "=>", entity["score"] )

```

If my order of labels changes, I get a different score for the detected entities. Why is this happening?

**Comment by urchade** [comment]
Hi, it is because the model uses positional encoding 
Bi-encode version of gliner of @Ingvarstep should not have this effect
